bundle/tests/scorecard/kuttl/service-account-name/00-assert.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sa-name-rc
+spec:
+  template:
+    spec:
+      serviceAccountName: sa-name-rc
+      serviceAccount: sa-name-rc
```

Comment:
`serviceAccount` is deprecated and will be removed in the future. It'll be good to avoid referencing it to avoid test failure in the future.